app/helpers/searches_helper.rb

```diff
@@ -1,12 +1,13 @@
 module SearchesHelper
-  def es_suggestions(gems)
-    return false if gems.size >= 1
-    return false unless gems.respond_to?(:response)
+  def search_suggestions(gems)
     suggestions = gems.response["suggest"]
-    return false if suggestions.blank?
-    return false if suggestions["suggest_name"].blank?
-    return false if suggestions["suggest_name"][0]["options"].empty?
+
+    options = suggestions["suggest_name"][0]["options"]
+    return if options.empty?
+
     suggestions.map { |_k, v| v.first["options"] }.flatten.map { |v| v["text"] }.uniq
+  rescue NoMethodError
```

Comment:
This is not good idea.  Raising and rescuing exception is expensive and is not common pattern.

This method looks ok as is to me, anyway if you would like to refactor, you can split it into two methods.

1. check for the input
2. retrieve needed info from input if it is possible